cli/src/semdep/parsers/poetry.py

```diff
@@ -147,7 +147,7 @@ class ValueLineWrapper:
     >> (poetry_dep | poetry_dep_extra | (string("package = []").result(None)))
     .sep_by(new_lines.optional())
     .map(lambda xs: [x for x in xs if x])
-    << string("\n").optional()
+    << new_lines.optional()
```

Comment:
So, the previous logic was accounting for a single optional newline, this one accounts for multiple.